server/sonar-server/src/test/java/org/sonar/server/computation/task/projectanalysis/issue/RuleRepositoryImplTest.java

```diff
@@ -29,6 +29,7 @@
 import org.sonar.api.rules.RuleType;
 import org.sonar.db.DbClient;
 import org.sonar.db.DbSession;
+import org.sonar.db.DbSessionImpl;
```

Comment:
Using `DbSessionImpl` is not required in this test. It can be replaced by `DbSession`.